build/nuke/Build.Steps.cs

```diff
@@ -304,15 +306,16 @@ private void RunBootstrappingTests()
 
     private void RunIntegrationTests()
     {
-        Project[] integrationTests = Solution
-            .GetProjects("IntegrationTests.*")
-            .ToArray();
+        Project[] integrationTests = Solution.GetIntegrationTests();
+
+        string filter = IsWin ? null : "WindowsOnly=true";
```

Comment:
I don't really understand this condition. Is this supposed to exclude IIS or Windows-only tests when being run on other OSes?

Reply:
Yep, this always happens with filter naming, depends if that filter is inclusive or exclusive. This filter here is exclusive, so it filters out tests with metadata `"WindowsOnly=true"`. Any ideas to improve readability are welcome.

Reply:
I thought these XUnit filters were inclusive. It looks like in the GitHub Actions runs for this PR, in the Ubuntu run for example, a lot of tests are not being run because it's only selecting tests with the "WindowsOnly=true" attribute. Maybe you could do the following

```suggestion
        string filter = IsWin ? null : "WindowsOnly!=true";
```

Reply:
My understand is if you add a trait it will only run the test if you explicitly set the trait. If the test doesn't have any trait it runs unless the test is filtered out. So it seems to me: 

1. Add the `[Trait("WindowsOnly", "True")]`
2. Set the filter to `WindowsOnly=True`

Is this incorrect?

Reply:
Fixed.